dali/util/nvml_wrap.cc

```diff
@@ -40,8 +41,23 @@ static nvmlReturn_t (*nvmlInternalDeviceGetCpuAffinity)(nvmlDevice_t device,
                                                         unsigned int cpuSetSize,
                                                         unsigned long* cpuSet);  // NOLINT(*)
 
+static nvmlReturn_t (*nvmlInternalDeviceGetBrand)(nvmlDevice_t device, nvmlBrandType_t *type);
+static nvmlReturn_t (*nvmlInternalDeviceGetCount_v2)(unsigned int *deviceCount);
+static nvmlReturn_t (*nvmlInternalDeviceGetHandleByIndex_v2)(unsigned int index,
+                                                             nvmlDevice_t *device);
+static nvmlReturn_t (*nvmlInternalDeviceGetCudaComputeCapability)(nvmlDevice_t device,
+                                                                  int *major, int *minor);
+
 static const char* (*nvmlInternalErrorString)(nvmlReturn_t r);
 
+namespace {
+bool is_driver_sufficient(int min_version) {
```

Comment:
why not have a `get_driver_version` and compare it yourself?

Reply:
`cudaDriverGetVersion` doesn't really return driver version, so `get_driver_version` wouldn't return driver version. It's better this way